README.md

```diff
@@ -1,89 +1,51 @@
-# Vet-Clinic-Database
- 
+# Vet_clinic database
+
+is a relational database used to create the database for a vet clinic. I created a table to store animals' information, insert some data into it, and query it.
+
```

Comment:
- [ ] Kindly remove this title and description from the the top because you already have it after Table of Contents 👍